src/lessons/08_grid_layout/grid-layout.slides.html

```diff
@@ -0,0 +1,758 @@
+                        grid-template-columns: auto auto auto;
+                        grid-template-rows: 80px 90px;
+                        justify-content: end
+                      }
+          
+                    # HTML
+                    <div class="grid-container">
+                      <div class="grid-item">Item 1</div>
+                      <div class="grid-item">Item 2</div>
+                      <div class="grid-item">Item 3</div>
+                      <div class="grid-item">Item 4</div>
+                      <div class="grid-item">Item 5</div>
+                    </div>
+            </code><br><div class="fragment grid-container grid-14">
+              <div class="grid-item">Item 1</div>
+              <div class="grid-item">Item 2</div>
+              <div class="grid-item">Item 3</div>
+              <div class="grid-item">Item 4</div>
+              <div class="grid-item">Item 5</div>
+          </div></pre>
+            </section>
+
+
+            <section>
+                <h1>Properties</h1>
+                <h3>Part 1: Parent Properties</h3>
+                <h3 class="fragment">Part 2: Child Properties</h3>
+            </section>
+
+            <section>
+                <h3>Item Location</h3>
+                <code>grid-column-start, grid-column-end, grid-row-start, grid-row-end</code>
+                <br>
+                <p><b>Values</b></p>
+                <ul>
+                    <li>line: number or name of grid line</li>
+                    <li>span &lt;number&gt;: item spans across given number of tracks</li>
+                    <li>span &lt;name&gt;: item spans until given track</li>
+                    <li>auto: auto-placement</li>
+                </ul>
+                <p class="fragment">Item will span 1, if no grid-column-end/grid-row-end is declared. Items can overlap each other. Use z-index to control stacking order.</p>
+            </section>
+
+            <section>
+                <h4>Example</h4>
+                <pre>
+              <code class="formattedCode">
+                # CSS
+                  .grid-container {
+                    grid-template-columns: [l1] 140px [l2] 150px [l3] auto [l4] 150px [l5] 140px [end];
+                    grid-template-rows: [r1-start] 25% [r1-end r2-start] 100px [r2-end r3-start] auto [r3-end];
+                  }
+
+                  .grid-item-1 {
+                    grid-column-start: 1;
+                    grid-column-end: span l4;
+                    grid-row-start: 2;
+                    grid-row-end: span 1;
+                  }
+      
+                # HTML
+                <div class="grid-container">
+                  <div class="grid-item-1">Item 1</div>
+                  <div class="grid-item-2">Item 2</div>
+                  <div class="grid-item-3">Item 3</div>
+                  <div class="grid-item-4">Item 4</div>
+                  <div class="grid-item-5">Item 5</div>
+                </div>
```

Comment:
Maybe is better to separate this into 2 block codes so the formatting is correct